bigframes/ml/preprocessing.py

```diff
@@ -282,8 +288,18 @@ class KBinsDiscretizer(
     def __init__(
         self,
         n_bins: int = 5,
+        strategy: Literal["uniform", "quantile", "kmeans"] = "quantile",
```

Comment:
Remove kmeans, if we don't have a plan to support.

Reply:
Done.